static/app/views/alerts/types.tsx

```diff
@@ -92,9 +97,8 @@ interface IssueAlert extends IssueAlertRule {
   latestIncident?: Incident | null;
 }
 
-interface MetricAlert extends MetricRule {
+export interface MetricAlert extends MetricRule {
   type: CombinedAlertType.METRIC;
-  latestIncident?: Incident | null;
```

Comment:
why remove latestIncident? This property is returned when passing the latestIncident expand parameter https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/list/rules/alertRulesList.tsx#L49 I assume the only reason this passes is because IssueAlert has latestIncident too which is probably not correct

Reply:
oh i see its available on MetricRule

Reply:
hmm that's probably wrong too lol

Reply:
Ah - no it's because the [MetricRule](https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/rules/metric/types.tsx#L124) that this is extending already includes the `latestIncident`